### DISCUSSION TITLE
OpenMMLab 2.0公测中文讨论区之MMSegmentation 1.x

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmsegmentation#2064] [General]
OpenMMLab 2.0公测中文讨论区之MMSegmentation 1.x

**Comment by Ttmx11** [comment]


**Reply by vansin** [reply]
确定是同一个数据集哇？

**Comment by shigejiafeimao** [comment]
如何使用publish_model.py

**Reply by vansin** [reply]
可能要晚一点时间来回答~

**Reply by RangeKing** [reply]
@shigejiafeimao 
`python tools/publish_model.py ${INPUT_FILENAME} ${OUTPUT_FILENAME}`
具体可以参考：https://mmsegmentation.readthedocs.io/zh_CN/latest/useful_tools.html#id2

**Comment by Ttmx11** [comment]
请问现在得到了网络预测结果图，以及真值图，如何使用mmseg内置的metric来评估iou,fscore等参数，（就是不使用model,config等文件）

**Reply by vansin** [reply]
可以试一下  MMEval 的 MeanIoU  
![img_v2_b887ec6f-8f27-47c6-a22e-8854a91e600g](https://user-images.githubusercontent.com/25839884/210790513-7797ef4c-5a92-4da5-8027-8529ac3bafb6.png)

**Reply by Ttmx11** [reply]
好的，谢谢，请问使用tools/test.py如何将图片保存成png格式，show-dir默认是jpg格式
